src/typed/ln-markets-api.d.ts

```diff
@@ -0,0 +1,173 @@
+// NOTE: The types here are incomplete. They have been copied from the following sources:
+// https://docs.lnmarkets.com/api/v1
+// https://npmjs.com/package/@ln-markets/api
+
+declare module '@ln-markets/api' {
```

Comment:
This helps us have types within tao-wallet. But it's not included in the build for clients to utilize. So we'll either need to adjust the build step later to include it or we'll need to publish the types to DefinitelyTyped.